### PR TITLE
fix(admin-ui): typos in text

### PR DESCRIPTION
**Pull request by gempain**
Fixes some typos found while reviewing translations.

Potential improvements in the future: fragmented translations may need merging with `<Trans>`. For example:

```
"analytics-preferences-disclaimer": "Pour créer la meilleure expérience de commerce électronique possible, nous aimerions obtenir des informations sur la manière dont vous utilisez Medusa. Les informations sur les utilisateurs nous permettent de construire des produits meilleurs, plus attrayants et plus utilisables. Nous ne collectons des données que pour améliorer le produit. Lisez quelles données nous recueillons dans notre",
"analytics-preferences-documentation": "documentation",
```

```
"help-dialog-feel-free-to-join-our-community-of": "Feel free to join our community of",
"help-dialog-merchants-and-e-commerce-developers": "merchants and e-commerce developers",
```

```
"upload-modal-drop-your-file-here-or": "Drop your file here, or",
"upload-modal-click-to-browse": "click to browse.",
```

Example from my previous PR with `sales-channels-display-available-count`:

```
<Trans
  i18nKey="sales-channels-display-available-count"
  availableChannelsCount={availableChannelsCount}
  totalChannelsCount={totalChannelsCount}
>
  Available in{" "}
  <span className="inter-base-semibold text-grey-90">
    {{ availableChannelsCount }}
  </span>{" "}
  out of{" "}
  <span className="inter-base-semibold text-grey-90">
    {{ totalChannelsCount }}
  </span>{" "}
  Sales Channels
</Trans>
```

### CODE REVIEW

File: packages/admin-ui/ui/src/i18n/index.ts [hunk 1]
```diff
@@ -10,6 +10,7 @@ void i18n
   // https://www.i18next.com/overview/configuration-options
   .init({
     fallbackLng: ["en"],
+    supportedLngs: ["en", "es"],
```
Comment (adrien2p) [inline packages/admin-ui/ui/src/i18n/index.ts]: suggestion: not sure it should be part of that pr

**Review by adrien2p — APPROVED** [review]
LGTM, just one comment

File: packages/admin-ui/ui/src/i18n/index.ts (hunk 1 above)
Comment (olivermrbl) [reply packages/admin-ui/ui/src/i18n/index.ts]: todo: It should not be part of the PR.

Comment (gempain) [reply packages/admin-ui/ui/src/i18n/index.ts]: Ouch, that's right, not sure what happened there.

**Comment by gempain** [comment]
Should be good to go.

**Pull request merged**